templates/manager/subscription-category-update.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html' %}
+{% block page-body %}
+<h1 class="heading-underline">Update Subscription Category</h1>
```

Comment:
Heading could use a `.h2` class to match sizing used elsewhere throughout Postmaster